Applications/SystemMonitor/main.cpp

```diff
@@ -559,7 +559,7 @@ NonnullRefPtr<GUI::Widget> build_graphs_tab()
         memory_graph.set_text_color(Color::Cyan);
         memory_graph.set_graph_color(Color::from_rgb(0x00bbbb));
         memory_graph.text_formatter = [](int value, int max) {
-            return String::format("%d / %d KB", value, max);
+            return String::format("%d / %d KiB", value, max);
```

Comment:
wait what?

> This keeps just "K" instead of "KiB"

> -KB, +KiB

Reply:
"This just keeps K in the table columns". This one was iirc what's on the "filesystems" tab, not a table.

Reply:
I'm not sure what's going on here, but the commit message seems a bit misleading/irrelevant to me. maybe that's just me though.

Reply:
I found it pretty clear, but I reworded it a bit, maybe it helps. Also fixed "captino" while here.

Reply:
I guess I was just in need of some zzz's. sorry for the trouble~